jgrapht-core/src/main/java/org/jgrapht/alg/cycle/PatonCycleBase.java

```diff
@@ -34,8 +39,8 @@
  *
  * @author Nikolay Ognyanov
  */
-public class PatonCycleBase<V, E> extends Cycle<V,E>
-        // was implementing UndirectedCycleBase
+public class PatonCycleBase<V, E>
```

Comment:
We already have a PR about the class `PatonCycleBase` which changes the interface.